main.go

```diff
@@ -226,3 +230,16 @@ func getNumJobsToKeep() int {
 	}
 	return numberOfJobsToKeep
 }
+
+func getDefaultSecurityContext() *corev1.SecurityContext {
+	return &corev1.SecurityContext{
```

Comment:
Note: this is just to stub out the behaviour as we codify using the context. We will need to capture the user input here or set to empty context.